kv/task.go

```diff
@@ -133,41 +134,23 @@ func (s *Service) findTaskByID(ctx context.Context, tx Tx, id influxdb.ID) (*inf
 		t.LatestCompleted = t.CreatedAt
 	}
 
-	// Attempt to fill in the owner ID based on the auth.
```

Comment:
note for reader:
https://github.com/influxdata/influxdb/blob/master/kv/migration/all/0003_task_owners.go#L15
This should not be nil anymore as a migration was delivered to backfill missing owners a while ago.